fuzz/fuzz_targets/streams.rs

```diff
@@ -1,13 +1,19 @@
 #![no_main]
 
+#[cfg(fuzzing)]
```

Comment:
The `cfg` statements here are noisy. Would you prefer a single `mod` behind a `cfg`?